packages/react/spec/imports/cjs-import.js

```diff
@@ -1,3 +1,3 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const { Provider } = require("../..");
+const { Provider } = require("@gadgetinc/react");
```

Comment:
I am not sure this is the same thing -- did you have to make this change?

Reply:
I don't recall making this change, I'll try reverting it.

Reply:
Right, so with "../.." the test fails with:

``` Command failed with exit code 1: node cjs-import.js
    node:internal/modules/cjs/loader:1137
      throw err;
      ^

    Error: Cannot find module '../..'
    Require stack:
    - /private/var/folders/4m/12k7sghx0n56pct6fz0zxf100000gn/T/js-clients-react-imports-z0kpAz/cjs-import.js
    ```
    
    However, looking at the other tests in context, I'm pretty sure requiring the main gadgetinc/react package is what this test is attempting to assert is functioning.
    
    I'm not sure I follow what "../.." was meant to represent in this case, but it seems to me that require("@gadgetinc/react") is more correct logically.

Reply:
Ok